### PR TITLE
Albums API integration

### PR DESCRIPTION
**Pull request by tagonzalez**
## Summary

Added request-promise package
Added albums API integration

## Trello Card

https://trello.com/c/uxTUE0Xa/34-integraci%C3%B3n-con-api-de-albumes-servicios-promises

### CODE REVIEW

File: app/controllers/albums.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+const listAlbums = (_, res) => {
+  albumsService
+    .listAlbums()
+    .then(albums => res.send({ albums: albums.map(snakeCaseKeys) }))
+    .catch(error => error);
+};
+
+const listAlbumPhotos = (req, res) => {
+  const albumId = req.params.id;
+  albumsService
```
Comment (arinaldi118) [inline app/controllers/albums.js]: You must add a `return` here

File: app/controllers/albums.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+const snakeCaseKeys = require('snakecase-keys');
+const albumsService = require('../services/albums');
+
+const listAlbums = (_, res) => {
+  albumsService
```
Comment (arinaldi118) [inline app/controllers/albums.js]: You must add a `return` here

File: app/controllers/albums.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    .listAlbums()
+    .then(albums => res.send({ albums: albums.map(snakeCaseKeys) }))
+    .catch(error => error);
```
Comment (arinaldi118) [inline app/controllers/albums.js]: You must send the error to errors middleware. The controller needs a third parameter to send things to the next middleware

File: app/controllers/albums.js (hunk 2 above)
Comment (tagonzalez) [reply app/controllers/albums.js]: Changed in latest commit!

File: app/controllers/albums.js (hunk 1 above)
Comment (tagonzalez) [reply app/controllers/albums.js]: Changed in latest commit!

File: app/controllers/albums.js (hunk 2 above)
Comment (tagonzalez) [reply app/controllers/albums.js]: Changed in latest commit!

**Pull request merged**